components/match2/wikis/mobilelegends/matchgroup_input_custom.lua

```diff
@@ -336,6 +336,8 @@ end
 
 function matchFunctions.getExtraData(match)
 	match.extradata = {
+		timezoneid = match.timezoneId,
+		timezoneoffset = match.timezoneOffset,
```

Comment:
Can we move this commons?

Reply:
not easily
we might want to look into default extradata and then append that in the /Customs
but for that we would probably also want to move other things over (e.g. comment)
so imo different PR

Reply:
`Module:Match._prepareMatchRecordForStore()`, one liner needed :)